README.md

```diff
@@ -94,6 +119,12 @@ If you were going to import the [h1 atoms/text pattern](https://github.com/patte
 
 `gulp clone --pattern node_modules/pattern-library/patterns/atoms/text/h1`
 
+
+## How to add your Github repo information to your project
+     
+     See **Generator Setup Steps** #2 above
```

Comment:
please make this a link:

See **[Generator Setup Steps](#generator-setup-steps)** #2 above